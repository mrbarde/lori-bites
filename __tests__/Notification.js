import React from 'react';
import Notification from '../src/notifications/components/Item';
import renderer from 'react-test-renderer';


test('Render a notification ', () => {
    const component = renderer.create(
        <Notification positionStyles={'topRight'} 
            delay={5000} message="Hello world!"/>
    );
    var tree = component.toJSON();
    expect(tree).toMatchSnapshot();
});
