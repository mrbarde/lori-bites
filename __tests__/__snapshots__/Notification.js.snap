// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render a notification  1`] = `
<li
  className="ln_notif_info"
  style={
    Object {
      "backgroundColor": "#ffffff",
      "border": "solid 1px #666666",
      "float": "right",
      "marginBottom": "20px",
      "padding": "30px 15px",
      "position": "relative",
      "width": "300px",
    }
  }
>
  <a
    className="ln_notif_close"
    href="javascript:;"
    style={
      Object {
        "color": "#999999",
        "fontSize": "14px",
        "position": "absolute",
        "right": "15px",
        "top": "10px",
      }
    }
  >
    Ã—
  </a>
  <p
    className="ln_notif_message"
    style={
      Object {
        "color": "#666666",
        "fontSize": "13px",
        "margin": "0px 0px",
      }
    }
  >
    Hello world!
  </p>
</li>
`;
